styles.css

```diff
@@ -0,0 +1,4 @@
+.title {
+    color: blueviolet;
+  }
+  
```

Comment:
- [ ] Great job here 💯 but I noticed that there is a `stylelint` error, please fix it .
All Linter result should be green ✅

![image](https://user-images.githubusercontent.com/45231276/198201832-51bb4cb0-289c-4412-a4e4-e80f8bce5aaf.png)

> 💡 This issue happened because of the white-space, you should format your CSS file correctly.

For Example
```
.title {
  color: blueviolet;
}
```
instead of 

```
.title {
    color: blueviolet;
  }

```